### ISSUE TITLE
issue

### ISSUE THREAD

**Issue by 18370029656** [dengyecode/hourglassattention#4]
Hello, I'm sorry to bother you. What does line 64 mean? Why not self.img_ truth = self.img
您好，不好意思打扰您了。请问第64行是什么含义？为什么不是self.img_truth = self.img
<img width="400" alt="image" src="https://user-images.githubusercontent.com/108872700/234748788-71bb18b6-2082-4393-b750-820a072b5243.png">

**Comment by dengyecode** [comment]
因为数据预处理的时候把图像的像素值转换的范围为[0,1]，而在我的模型中处理的像素值范围为[-1,1]，因此通过该行代码来转换图像的像素值

**Comment by 18370029656** [comment]
thank you for your answer. May I ask if the data preprocessing operation needs to be done by oneself or included in the code? I don't seem to have found any normalization operation in the code

**Comment by ScarletBlaze** [comment]
you can find the related code in the "dataloader/data_loader.py". In fact, the normalization code is the "transforms.ToTensor()". You can read the doucument of pytorch to know how it works.
